### PR TITLE
test: update jest cli options

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by linhf123 — APPROVED** [review]

**Review by yoyo837 — APPROVED** [review]

**Review by li-jia-nan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -87,13 +87,13 @@
     "sort": "npx sort-package-json",
     "sort-api": "antd-tools run sort-api-table",
     "start": "cross-env PORT=8001 dumi dev",
-    "test": "jest --config .jest.js --cache=false",
-    "test:update": "jest --config .jest.js --cache=false -u",
+    "test": "jest --config .jest.js --no-cache --silent",
```
[the remark this answers is not in the document]
Comment (afc163) [reply package.json:90]: 现在无效 console 太多了，基本上也看不到。

Comment (afc163) [inline package.json:90]:
```suggestion
    "test": "jest --config .jest.js --no-cache",
```

File: package.json [hunk 2]
```diff
@@ -87,13 +87,13 @@
...
+    "test:update": "jest --config .jest.js --no-cache -u",
     "test-all": "sh -e ./scripts/test-all.sh",
-    "test-node": "npm run version && jest --config .jest.node.js --cache=false",
+    "test-node": "npm run version && jest --config .jest.node.js --no-cache --silent",
```
Comment (afc163) [inline package.json:93]:
```suggestion
    "test-node": "npm run version && jest --config .jest.node.js --no-cache",
```

File: package.json [hunk 3]
```diff
@@ -87,13 +87,13 @@
...
     "tsc": "tsc --noEmit",
-    "site:test": "jest --config .jest.site.js --cache=false --force-exit",
-    "test-image": "jest --config .jest.image.js --no-cache -i -u",
+    "site:test": "jest --config .jest.site.js --no-cache --force-exit",
+    "test-image": "jest --config .jest.image.js --no-cache -i -u --silent",
```
Comment (afc163) [inline package.json:96]:
```suggestion
    "test-image": "jest --config .jest.image.js --no-cache -i -u",
```

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (afc163) [reply package.json]: 那我先把 --slient 去掉。

**Pull request merged**